doc/litecoin-release-notes/release-notes-0.21.1.md

```diff
@@ -0,0 +1,1200 @@
+- #19018 fixing description of the field sequence in walletcreatefundedpsbt RPC method (limpbrains)
+- #19367 Span pitfalls (sipa)
+- #19408 Windows WSL build recommendation to temporarily disable Win32 PE support (sipsorcery)
+- #19407 explain why passing -mlinker-version is required when cross-compiling (fanquake)
+- #19452 afl fuzzing comment about afl-gcc and afl-g++ (Crypt-iQ)
+- #19258 improve subtree check instructions (Sjors)
+- #19474 Use precise permission flags where possible (MarcoFalke)
+- #19494 CONTRIBUTING.md improvements (jonatack)
+- #19268 Add non-thread-safe note to FeeFilterRounder::round() (hebasto)
+- #19547 Update macOS cross compilation dependencies for Focal (hebasto)
+- #19617 Clang 8 or later is required with `FORCE_USE_SYSTEM_CLANG` (fanquake)
+- #19639 Remove Reference Links #19582 (RobertHosking)
+- #19605 Set `CC_FOR_BUILD` when building on OpenBSD (fanquake)
+- #19765 Fix getmempoolancestors RPC result doc (MarcoFalke)
+- #19786 Remove label from good first issue template (MarcoFalke)
+- #19646 Updated outdated help command for getblocktemplate (jakeleventhal)
+- #18817 Document differences in bitcoind and bitcoin-qt locale handling (practicalswift)
+- #19870 update PyZMQ install instructions, fix `zmq_sub.py` file permissions (jonatack)
+- #19903 Update build-openbsd.md with GUI support (grubles)
+- #19241 help: Generate checkpoint height from chainparams (luke-jr)
+- #18949 Add CODEOWNERS file to automatically nominate PR reviewers (adamjonas)
+- #20014 Mention signet in -help output (hebasto)
+- #20015 Added default signet config for linearize script (gr0kchain)
+- #19958 Better document features of feelers (naumenkogs)
+- #19871 Clarify scope of eviction protection of outbound block-relay peers (ariard)
+- #20076 Update and improve files.md (hebasto)
+- #20107 Collect release-notes snippets (MarcoFalke)
+- #20109 Release notes and followups from 19339 (glozow)
+- #20090 Tiny followups to new getpeerinfo connection type field (amitiuttarwar)
+- #20152 Update wallet files in files.md (hebasto)
+- #19124 Document `ALLOW_HOST_PACKAGES` dependency option (skmcontrib)
+- #20271 Document that wallet salvage is experimental (MarcoFalke)
+- #20281 Correct getblockstats documentation for `(sw)total_weight` (shesek)
+- #20279 release process updates/fixups (jonatack)
+- #20238 Missing comments for signet parameters (decryp2kanon)
+- #20756 Add missing field (permissions) to the getpeerinfo help (amitiuttarwar)
+- #20668 warn that incoming conns are unlikely when not using default ports (adamjonas)
+- #19961 tor.md updates (jonatack)
+- #19050 Add warning for rest interface limitation (fjahr)
+- #19390 doc/REST-interface: Remove stale info (luke-jr)
+- #19344 docs: update testgen usage example (Bushstar)
+
+Credits
+=======
+
+Thanks to everyone who directly contributed to this release:
+
+- [The Bitcoin Core Developers](https://github.com/bitcoin/bitcoin/tree/master/doc/release-notes)
```

Comment:
Litecoin Developers?

Reply:
No, this is saying the bitcoin developers plus the few litecoin devs underneath.